azext_iot/_help.py

```diff
@@ -820,6 +820,30 @@
         --metric-type system
 """
 
+helps[
+    "iot hub configuration test-queries"
+] = """
+    type: command
+    short-summary: Validates target condition and custom metric queries for a configuration on the IoT Hub
+    long-summary: |
+      This command supports both inline json and file input for custom metric queries `--custom-metric-queries`.
+      Review examples and parameter description for details on how to fully utilize the operation.
+
+      Read more about using quotation marks and escapes characters in different shells here:
+      https://github.com/Azure/azure-iot-cli-extension/wiki/Tips#use-quotation-mark-properly-for-command-continuation
+
+    examples:
+    - name: Validate the target condition and custom metric queries with inline json.
+      text: >
+        az iot hub configuration test-queries -n {iothub_name} --target-condition "from devices.modules where tags.building=9"
+        --custom-metric-queries '{\\"mymetric\\":""select moduleId from devices.modules where tags.location=''US''"",
```

Comment:
json input by file is fine, but have we thought about alternative to inline json ways to take user input to reduce the users need for dealing with shell quotation issues? I know the edge deployment create command has a similar pattern to this (whose implementation is legacy) , but I'm thinking of alternatives like an nargs kvp style input, something like:

```
az iot hub configuration test-queries \
-n myiothub \
--target-condition "from devices.modules where tags.building=9" \
--custom-metric-queries mymetric1="select moduleId from devices.modules where tags.location='US'" mymetric2="select moduleId from devices.modules where tags.location='EU'" 
```

Thoughts?

Reply:
I agree this is much more clear and easy to use, let me dive into how to implement these, thanks for the insight!

Reply:
We can also think about (in a separate task) retrofitting the existing commands that support custom metric queries with this pattern for consistency.